test/unit/util/test_matrix_policy.hpp

```diff
@@ -204,6 +205,32 @@ MatrixPolicy<double> testAssignmentFromVector()
   return matrix;
 }
 
+template<template<class> class MatrixPolicy>
+MatrixPolicy<double> testAxpy()
+{
+  MatrixPolicy<double> matrix{ 4, 3, 100.0 };
+  MatrixPolicy<double> other{ 4, 3, 200.0 };
+  double alpha = 1.39;
+  double sum = 0.0;
+  double result = 0.0;
+
+  for (int i = 0; i < 4; ++i)
+    for (int j = 0; j < 3; ++j)
+    {
+      matrix[i][j] = i * 10.3 + j * 100.5;
+      other[i][j] = i * 1.7 + j * 10.2;
+      sum += i * 10.3 + j * 100.5 + alpha * (i * 1.7 + j * 10.2);
```

Comment:
```suggestion
      auto y = i * 10.3 + j * 100.5;
      auto x = i * 1.7 + j * 10.2;
      matrix[i][j] = y
      other[i][j] = x;
      sum += y + alpha * x;
```
This might be a little clearer and saves us from double computing x and y without needing to rely on the accessors for `matrix` and `other`.

Reply:
I like this, thanks!